packages/amazonq/test/e2e_new/amazonq/utils/generalUtils.ts

```diff
@@ -39,6 +64,69 @@ export async function waitForElements(webview: WebviewView, locator: By, timeout
     return await webview.findWebElements(locator)
 }
 
+/**
+ * Presses a single key globally
+ * @param driver The WebDriver instance
+ * @param key The key to press
+ */
+export async function pressKey(driver: WebDriver, key: keyof typeof Key): Promise<void> {
+    await driver.actions().sendKeys(key).perform()
+}
+
+type OSType = 'windows' | 'mac'
+type modifierKey = 'alt' | 'ctrl' | 'shift' | 'option' | 'command'
+
+const MODIFIERKEYS: Record<OSType, Record<modifierKey, string>> = {
+    windows: {
+        alt: Key.ALT,
+        ctrl: Key.CONTROL,
+        shift: Key.SHIFT,
+        option: Key.ALT,
+        command: Key.COMMAND,
+    },
+    mac: {
+        alt: Key.ALT,
+        ctrl: Key.CONTROL,
+        shift: Key.SHIFT,
+        option: Key.ALT,
+        command: Key.COMMAND,
+    },
+}
+
+/**
+ * Presses a keyboard shortcut with modifier keys
+ * @param driver The WebDriver instance
+ * @param key The key to press
+ * @param modifier The modifier key(s)
+ * @param osType The operating system type
+ */
+export async function pressShortcut(
+    driver: WebDriver,
+    key: string,
+    modifier: modifierKey | modifierKey[],
+    osType: OSType = 'windows'
+): Promise<void> {
+    const actions = driver.actions({ bridge: true })
+
+    try {
+        const modifiers = Array.isArray(modifier) ? modifier : modifier ? [modifier] : []
+        for (const mod of modifiers) {
+            if (!(mod in MODIFIERKEYS[osType])) {
+                throw new Error(`Invalid modifier key '${mod}' for ${osType}`)
+            }
+            actions.keyDown(MODIFIERKEYS[osType][mod])
+        }
+        actions.sendKeys(key)
```

Comment:
what does `sendKeys` do here? It looks like we press the key earlier, and release it at the end, so why do we also need to send it?